src/components/Shifting/BoardView.tsx

```diff
@@ -35,16 +36,16 @@ export default function BoardView() {
   });
 
   const [modalFor, setModalFor] = useState<{
-    externalId?: string;
+    external_id?: string | undefined;
```

Comment:
```suggestion
    external_id?: string;
```
a `?` indirectly means `undefined`

Reply:
Yes, thanks for the suggestion, I removed it